R/plots.R

```diff
@@ -62,7 +62,10 @@ plotObservationsCM <- function(...) {
 plotCM <- function(flux.or.conc, show.observations, load.model, finalloads, observations=NULL, 
                    composite=TRUE, linear.interpolation=FALSE, regression=TRUE,  xrange="none", 
                    dateField="Date", verbose=FALSE) {
-  
+  #silly thing needed to pass R CMD check
+  # from http://stackoverflow.com/questions/8096313/no-visible-binding-for-global-variable-note-in-r-cmd-check
+  aes_string=geom_point=yday=xlab=NULL
```

Comment:
this is fixed here https://github.com/miguelcleon/loadflex/blob/master/R/plots.R#L66